Sources/JSONMatcher/Builder.swift

```diff
@@ -14,13 +14,13 @@ struct Builder {
         }
     }
 
-    private func buildRawJSONElement<T>(object: T) -> BaseElementType {
+    fileprivate func buildRawJSONElement<T>(_ object: T) -> BaseElementType {
```

Comment:
I made all `fileprivate` methods and properties into `private`, and things still work.